src/super_gradients/common/sg_loggers/wandb_sg_logger.py

```diff
@@ -102,7 +105,11 @@ def __init__(
                     )
                 wandb_id = self._get_wandb_id()
 
-        run = wandb.init(project=project_name, name=experiment_name, entity=entity, resume=resumed, id=wandb_id, **kwargs)
+        run = (
+            wandb.init(project=project_name, name=experiment_name, entity=entity, resume=resumed, id=wandb_id, sync_tensorboard=sync_tensorboard, **kwargs)
```

Comment:
why do we need to synch tensorboard if all information is logged to wandb? Wouldn't this create duplicated data?